storage/README.md

```diff
@@ -2,30 +2,78 @@
 
 Entities describe a business entity persisted to storage layer.
 
-## How to create entity
+## How to use
+
+The initial implementation includes CRUD operations for the entities `Account`, `Block` and `Transaction`.
```

Comment:
Why are the other entities (Round, Peer, Migration) not listed here?

Reply:
Because they will be only available inside chain module